### ISSUE TITLE
"AFEW dataset face alignment" and "accuracy on validation" problem.

### ISSUE THREAD

**Issue by yuexihang** [Open-Debin/Emotion-FAN#32]
Hello. I use the **./data/face_alignment_code/frame2face_afew.py** to process the frame files from AFEW videos.
However, I found some face can't be detected, which causes  that some aligned face folder was empty.

I want to know about whether you just use the **./data/face_alignment_code/frame2face_afew.py** to process or not? Or do you have another ways?

What'more, I got **the overfitting error** after just several epochs, and the accuracy on validation is just around 41 (without the relation-attention) or 16 (with the relation-attention). 

I use the code without any changing.

**Comment by yuexihang** [comment]
I have solved the problem.

**Comment by ChaoWANG0511** [comment]
Hello yuexihang, I meet the same **overfitting error** problem, could you please give a hint about how you solved the problem? thank you very much :)

**Comment by yuexihang** [comment]
> Hello yuexihang, I meet the same **overfitting error** problem, could you please give a hint about how you solved the problem? thank you very much :)

Hello, I don't know if you have solved it. 
I got the **overfitting error** for an interesting reason. Human face was not detected on some frames by the preprocessing method and I just choosed to ignore these frames, which causes **the disalignment between emotion labels and frames**. It means that a lot of frames have the incorrect emotion label.

**Comment by wuyanzu** [comment]
> > Hello yuexihang, I meet the same **overfitting error** problem, could you please give a hint about how you solved the problem? thank you very much :)
> 
> Hello, I don't know if you have solved it. I got the **overfitting error** for an interesting reason. Human face was not detected on some frames by the preprocessing method and I just choosed to ignore these frames, which causes **the disalignment between emotion labels and frames**. It means that a lot of frames have the incorrect emotion label.
 
Hello, I'm new to this, could you let me know how you modified the code?

**Comment by Open-Debin** [comment]
> > Hello yuexihang, I meet the same **overfitting error** problem, could you please give a hint about how you solved the problem? thank you very much :)
> 
> Hello, I don't know if you have solved it. I got the **overfitting error** for an interesting reason. Human face was not detected on some frames by the preprocessing method and I just choosed to ignore these frames, which causes **the disalignment between emotion labels and frames**. It means that a lot of frames have the incorrect emotion label.

Thank you very much that telling me the reason for the relation-attention. The old version will not have this kind of problem https://github.com/Open-Debin/Emotion-FAN/releases/tag/v1.0_august_2019. Also, the model can not be trained from scratch. please train it based on pretrained emotion model. Due to different Machines, I suggest you try different learning rates.

**Comment by fw-codes-files** [comment]
> Hello. I use the **./data/face_alignment_code/frame2face_afew.py** to process the frame files from AFEW videos. However, I found some face can't be detected, which causes that some aligned face folder was empty.
> 
> I want to know about whether you just use the **./data/face_alignment_code/frame2face_afew.py** to process or not? Or do you have another ways?
> 
> What'more, I got **the overfitting error** after just several epochs, and the accuracy on validation is just around 41 (without the relation-attention) or 16 (with the relation-attention).
> 
> I use the code without any changing.

May I ask you for AFEW dataset? I sent the e-mail to the organization but they didn't reply me
